.github/actions/upstream-test/action.yml

```diff
@@ -40,14 +40,34 @@ runs:
           username: ${{ github.repository_owner }}
           password: ${{ inputs.token }}
 
+      - name: Configure docker image tag
+        shell: bash -l {0}
+        run: |
+          image_base_tag=ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ inputs.ubuntu_version }}-${{ inputs.compiler}}-hdf5_${{ inputs.hdf5_version}}-moab_${{ inputs.moab_version }}
+          if [ ${{ inputs.geant4_version }} != "OFF" ] ; then
+            image_base_tag=${image_base_tag}-geant4_${{ inputs.geant4_version }}
+            GEANT4=ON
+          else
+            GEANT4=OFF
+          fi
+          if [ ${{ inputs.double_down_version }} != "OFF" ] ; then
+            image_base_tag=${image_base_tag}-double_down_${{ inputs.double_down_version }}
+            DOUBLE_DOWN=ON
+          else
+            DOUBLE_DOWN=OFF
+          fi
+          echo "image_base_tag=${image_base_tag}" >> $GITHUB_ENV
+          echo "GEANT4=${GEANT4}" >> $GITHUB_ENV
+          echo "DOUBLE_DOWN=${DOUBLE_DOWN}" >> $GITHUB_ENV
```

Comment:
I think we should always tag the image with `OFF` if it's off or the version number if it's on.  This will remove the need for all of this.

Reply:
We can also then use the logic of the Dockerfile to handle the ON vs OFF case

Reply:
You are saying instead of setting ON/OFF here, we should make this method in the Docker file? This ON value is required for building DAGMC with Geant4 and DD. That means we need these conditions sooner or later.